.github/PULL_REQUEST_TEMPLATE/bug_fix.md

```diff
@@ -1,23 +1,18 @@
----

```

Comment:
Originally, I think you were going to delete these existing, specialized PR templates. Did you learn something that changed your mind?

Reply:
I guess I'm just hopeful GitHub will eventually support this. But you're right, probably should delete in meantime as they will just get out of date.

Though see comment about the release.md one. Think that would be used? Or just get rid?

Reply:
Thought is over and they are gone.

Reply:
No strong opinion either way about the PR templates, just curious. Reviewing this again now...